contracts/OrderBook.sol

```diff
@@ -67,6 +67,8 @@ abstract contract OrderBook is CloberOrderBook, ReentrancyGuard, RevertOnDelegat
     mapping(address => uint256) public override uncollectedHostFees;
     mapping(address => uint256) public override uncollectedProtocolFees;
     mapping(uint256 => Order) private _orders;
+    uint16 public override blockTradeLogIndex;
+    BlockTradeLog[65535] private _blockTradeLogs;
```

Comment:
```suggestion
    BlockTradeLog[type(uint16).max-1] private _blockTradeLogs;
```

Reply:
No need to subtract 1. `type(uint16).max == 65535`

Reply:
Also, `BlockTradeLog[type(uint16).max] private _blockTradeLogs;` this expression is invalid at Solidity.